### PR TITLE
Feature/#172 account billing

### PR DESCRIPTION
**Pull request by morcosan**

### CODE REVIEW

File: .tooling/eslint/custom-plugin.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ export const customPlugin = {
 					const isDS = path.startsWith('@ds/') || path.includes('../src-ds/')
 					const isI18n = path.startsWith('@i18n/') || path.includes('../src-i18n/')
 					const isUtils = path.startsWith('@utils/') || path.includes('../src-utils/')
+					const isAppLib = path.startsWith('@app/library') || path.includes('../src-app/library/')
```
Comment (costinEEST) [inline .tooling/eslint/custom-plugin.js:13]: `path.includes('/library/')` ?

File: src-api/docs/_page.tsx [hunk 2]
```diff
@@ -54,7 +71,7 @@ const ApiDocsPage = () => {
 
 	const onClickLogo = (event: ReactMouseEvent) => {
 		event.preventDefault()
-		window.location.assign('/')
+		window.location.assign(ENV__ROOT_URL_PATH + '/') // Forced refresh
```
Comment (costinEEST) [inline src-api/docs/_page.tsx:74]: Let's do it the React's way - https://reactrouter.com/en/main/components/link#reloaddocument

Comment (morcosan) [reply src-api/docs/_page.tsx:74]: I was actually wondering if there is already a built-in feature.

File: .tooling/eslint/custom-plugin.js (hunk 1 above)
Comment (morcosan) [reply .tooling/eslint/custom-plugin.js:13]: That's too generic, it might cause issues.

**Pull request merged**